index.html

```diff
@@ -1,140 +1,170 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<title>Git for Windows</title>
-		<meta name="description" content="We bring the awesome Git VCS to Windows">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
-		<link rel="stylesheet" href="css/pack.css">
-		<link rel="shortcut icon" href="favicon.ico">
-		<!--[if lt IE 9]>
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Git for Windows</title>
+	<meta name="description" content="We bring the awesome Git VCS to Windows">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="robots" content="all,follow">
+	<!-- Bootstrap CSS-->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/css/bootstrap.min.css">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
+	<!-- Custom Stylesheet -->
+	<link rel="shortcut icon" href="favicon.ico">
+	<link rel="stylesheet" href="css/custom.css">
+	<!--[if lt IE 9]>
 		<script src="js/html5shiv.js"></script>
 		<script src="js/respond.js"></script>
 		<![endif]-->
-	</head>
-	<body>
-		<header>
-			<nav class="content">
-				<a href="index.html"><h1 class="gittext lowercase">Git<span>for Windows</span></h1></a>
-				<div class="version"><a href="https://github.com/git-for-windows/git/releases/tag/v2.29.2.windows.2" title="Version 2.29.2.2 was published on Wed, 04 Nov 2020 23:57:14 UTC">Version 2.29.2.2</a></div>
```

Comment:
This `"version"` line is very special: it is [used by the automation after new Git for Windows versions are released](https://github.com/git-for-windows/git-for-windows.github.io/blob/a169c878fb7938a43f265081c1add83c331a549e/bump-version.js#L16). Removing this line will break the automation.

Reply:
Hmmm, this seemed way out of place in the original design, and I found most projects do not simply display that and point to the latest releases, but, in this case, since, it's important, I'll revisit this.